_topic_map.yml

```diff
@@ -1182,84 +1166,112 @@ Topics:
-  - Name: Migrating a virtual machine instance to another node
-    File: cnv-migrate-vmi
-  - Name: Monitoring live migration of a virtual machine instance
-    File: cnv-monitor-vmi-migration
-  - Name: Cancelling the live migration of a virtual machine instance
-    File: cnv-cancel-vmi-migration
-  - Name: Configuring virtual machine eviction strategy
-    File: cnv-configuring-vmi-eviction-strategy
+  - Name: Live migration
+    Dir: cnv-live-migration
+    Topics:
+    - Name: Virtual machine live migration
+      File: cnv-live-migration
+    - Name: Live migration limits and timeouts
+      File: cnv-live-migration-limits
+    - Name: Migrating a virtual machine instance to another node
+      File: cnv-migrate-vmi
+    - Name: Monitoring live migration of a virtual machine instance
+      File: cnv-monitor-vmi-migration
+    - Name: Cancelling the live migration of a virtual machine instance
+      File: cnv-cancel-vmi-migration
+    - Name: Configuring virtual machine eviction strategy
+      File: cnv-configuring-vmi-eviction-strategy
 # Node maintenance mode
-  - Name: Node maintenance mode
-    File: cnv-node-maintenance
-  - Name: Setting a node to maintenance mode
-    File: cnv-setting-node-maintenance
-  - Name: Resuming a node from maintenance mode
-    File: cnv-resuming-node
-# Installing VirtIO drivers on Windows virtual machines
-  - Name: Installing VirtIO driver on an existing Windows virtual machine
-    File: cnv-installing-virtio-drivers-on-existing-windows-vm
-  - Name: Installing VirtIO driver on a new Windows virtual machine
-    File: cnv-installing-virtio-drivers-on-new-windows-vm
+  - Name: Node maintenance
+    Dir: cnv-node-maintenance
+    Topics:
+    - Name: Node maintenance mode
+      File: cnv-node-maintenance
+    - Name: Setting a node to maintenance mode
+      File: cnv-setting-node-maintenance
+    - Name: Resuming a node from maintenance mode
+      File: cnv-resuming-node
 # Logging, events, and monitoring
-  - Name: Viewing logs
-    File: cnv-logs
-  - Name: Viewing events
-    File: cnv-events
-    Name: Viewing cluster information
-    File: cnv-using-dashboard-to-get-cluster-info
-  - Name: OpenShift cluster monitoring, logging, and Telemetry
-    File: cnv-openshift-cluster-monitoring
-  - Name: Collecting container-native virtualization data for Red Hat Support
-    File: cnv-collecting-cnv-data
+  - Name: Logging, events, and monitoring
+    Dir: cnv-logging-events-monitoring
+    Topics:
+    - Name: Viewing logs
+      File: cnv-logs
+    - Name: Viewing events
+      File: cnv-events
+      Name: Viewing cluster information
+      File: cnv-using-dashboard-to-get-cluster-info
+    - Name: OpenShift cluster monitoring, logging, and Telemetry
+      File: cnv-openshift-cluster-monitoring
+    - Name: Collecting container-native virtualization data for Red Hat Support
+      File: cnv-collecting-cnv-data
 - Name: Container-native virtualization 2.1 release notes
```

Comment:
I'd probably move the release notes assembly out of this subdirectory and to the top of the CNV bucket, just before installation.

Reply:
Sounds good.